Python/instrumentation.c

```
@@ -1236,6 +1236,7 @@ _Py_call_instrumentation_jump(
            event == PY_MONITORING_EVENT_BRANCH_RIGHT ||
            event == PY_MONITORING_EVENT_BRANCH_LEFT);
     int to = (int)(dest - _PyFrame_GetBytecode(frame));
+    assert(to <= INT_MAX / (int)sizeof(_Py_CODEUNIT));
```

If there is a real risk that this case occurs, I would prefer to raise an exception.

---

Do you mean something like this?
```c
if (to <= INT_MAX / (int)sizeof(_Py_CODEUNIT)) {
    PyErr_SetString(PyExc_OverflowError, "instruction offset is too large for int");
    return NULL;
}
```

---

Yes

---

Okay, added
https://github.com/python/cpython/pull/135202

---

I will close this one request